crates/core/tedge/src/cli/refresh_bridges.rs

```diff
@@ -28,19 +29,38 @@ impl Command for RefreshBridgesCmd {
     fn execute(&self) -> anyhow::Result<()> {
         let clouds = established_bridges(&self.config_location);
 
-        if clouds.is_empty() {
+        if clouds.is_empty() && !self.config.mqtt.bridge.built_in {
             println!("No bridges to refresh.");
             return Ok(());
         }
 
         let common_mosquitto_config = CommonMosquittoConfig::from_tedge_config(&self.config);
         common_mosquitto_config.save(&self.config_location)?;
 
-        for cloud in clouds {
-            println!("Refreshing bridge {cloud}");
+        if !self.config.mqtt.bridge.built_in {
+            for cloud in &clouds {
+                println!("Refreshing bridge {cloud}");
 
-            let bridge_config = super::connect::bridge_config(&self.config, cloud)?;
-            refresh_bridge(&bridge_config, &self.config_location)?;
+                let bridge_config = super::connect::bridge_config(&self.config, *cloud)?;
+                refresh_bridge(&bridge_config, &self.config_location)?;
+            }
+        }
+
+        for cloud in [Cloud::Aws, Cloud::Azure, Cloud::C8y] {
+            // (attempt to) reassert ownership of the certificate and key
+            // This is necessary when upgrading from the mosquitto bridge to the built-in bridge
```

Comment:
We will have to find a more dynamic way to handle certificate ownership. But that's okay for now.